test/functional/README.md

```diff
@@ -34,7 +34,7 @@ don't have test cases for.
 
 - Name the test `<area>_<name>.py`, where `<name>` is a keyword describing your test
   and `<area>` can be one of the following:
-    - `esperanza` for tests for esperanza Proof-of-Stake features, eg `esperanza_finalization.py`
+    - `finalization` for tests that check the finalization functionality, eg `finalization_finalization.py`
```

Comment:
There is no esperanza_finalization.py at the moment.

Reply:
I will put another example, this is a result of a find and replace.